### PR TITLE
More overrides slimming

### PR DESCRIPTION
**Pull request by kaspth**
This is a work in progress, where I've been exploring some more slimming. I'm doing other things now, so I'll get back to these in a bit.

### CODE REVIEW

**Comment by newstler** [comment]
@kaspth How's that going? So many commits!

**Comment by kaspth** [comment]
@newstler I haven't gotten back to Jbuilder::Schema in a bit, I'm hoping to come back soon and wrap up my open PRs. Just to be clear, is there anything you're specifically waiting on from me?

**Comment by newstler** [comment]
@kaspth Not really, just was worried that such a huge work might become abandoned 😅

File: test/jbuilder/schema/builder_test.rb [hunk 1]
```diff
@@ -54,20 +54,14 @@ class Jbuilder::Schema::BuilderTest < ActiveSupport::TestCase
...
-            type: :array,
-            items: {
-              "id" => {type: :integer},
-              "title" => {type: :string}
-            }
+        "articles" => {
+          type: :array,
+          items: {
+            "id" => {type: :integer},
+            "title" => {type: :string}
```
Comment (kaspth) [inline test/jbuilder/schema/builder_test.rb:64]: @newstler do you happen to know which is the more correct structure here? The template is just `json.articles @articles, :id, :title` and I think the new test is more accurate.

I'll merge for now, and then I can revisit.

**Pull request merged**